packages/grafana-ui/src/components/TimePicker/TimeRangeInput.story.tsx

```diff
@@ -1,67 +1,122 @@
+import { Story } from '@storybook/react';
 import React from 'react';
 import { action } from '@storybook/addon-actions';
-import { dateTime, TimeFragment } from '@grafana/data';
+import { dateTime, DefaultTimeZone, TimeRange, TimeZone } from '@grafana/data';
 import { withCenteredStory } from '../../utils/storybook/withCenteredStory';
 import { UseState } from '../../utils/storybook/UseState';
 import { TimeRangeInput } from '@grafana/ui';
+import { TimeRangeInputProps } from './TimeRangeInput';
 import mdx from './TimeRangeInput.mdx';
 
 export default {
   title: 'Pickers and Editors/TimePickers/TimeRangeInput',
   component: TimeRangeInput,
   decorators: [withCenteredStory],
+  argTypes: {
+    hideTimeZone: {
+      control: 'boolean',
+    },
+    placeholder: {
+      control: 'text',
+    },
+    clearable: {
+      control: 'boolean',
+    },
+    isReversed: {
+      control: 'boolean',
+    },
+    hideQuickRanges: {
+      control: 'boolean',
+    },
+  },
   parameters: {
     docs: {
       page: mdx,
     },
   },
 };
 
-export const basic = () => {
-  return (
-    <UseState
-      initialState={{
-        from: dateTime(),
-        to: dateTime(),
-        raw: { from: 'now-6h' as TimeFragment, to: 'now' as TimeFragment },
-      }}
-    >
-      {(value, updateValue) => {
-        return (
-          <TimeRangeInput
-            value={value}
-            onChange={timeRange => {
-              action('onChange fired')(timeRange);
-              updateValue(timeRange);
-            }}
-          />
-        );
-      }}
-    </UseState>
+const args = {
+  hideTimeZone: false,
+  placeholder: undefined,
+  clearable: false,
+  isReversed: true,
+  hideQuickRanges: false,
+};
+
+interface State {
+  value: TimeRange;
+  timeZone: TimeZone;
+}
+
+const getComponentWithState = (initialState: State, props: TimeRangeInputProps) => (
+  <UseState initialState={initialState}>
+    {(state, updateValue) => {
+      return (
+        <TimeRangeInput
+          {...props}
+          value={state.value}
+          timeZone={state.timeZone}
+          onChange={value => {
+            action('onChange fired')(value);
+            updateValue({
+              ...state,
+              value,
+            });
+          }}
+          onChangeTimeZone={timeZone => {
+            action('onChangeTimeZone fired')(timeZone);
+            updateValue({
+              ...state,
+              timeZone,
+            });
+          }}
+        />
+      );
+    }}
+  </UseState>
+);
+
+export const relative: Story<TimeRangeInputProps> = props => {
```

Comment:
```suggestion
export const Relative: Story<TimeRangeInputProps> = props => {
```
We're moving to a format where story names are uppercased, altho there are still some stories where we haven't made these changes.